fe/fe-core/pom.xml

```diff
@@ -566,6 +566,18 @@ under the License.
             <artifactId>hadoop-aws</artifactId>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure-datalake -->
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-azure-datalake</artifactId>
```

Comment:
I think you need to check if there are some dependency conflicts and exclude them

Reply:
I've run pr regression, it works fine.